packages/commonwealth/client/scripts/views/pages/view_thread/index.tsx

```diff
@@ -519,7 +519,7 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
               label: 'Delete',
               iconLeft: 'trash' as const,
               onClick: async (e) => {
-                e.preventDefault();
+                // e.preventDefault();
```

Comment:
works in thread view but dose not work in 'thread preview menu'
![image](https://user-images.githubusercontent.com/14819225/223739621-07e6dc27-86d8-4efa-9a51-73f210d5d7b6.png)

Reply:
Addressed